src/js/components/DataChart/DataChart.js

```diff
@@ -527,7 +527,7 @@ const DataChart = forwardRef(
       series.forEach(({ property, render }) => {
         if (
           !render &&
-          data.length > 1 &&
+          data.length >= 1 &&
```

Comment:
```suggestion
          data.length > 1 &&
```
I think we should revert this change back. If we only have 1 data point we don't need to the createDateFormat code